Azure/sonic-buildimage
# [Marvell]Update driver tag to pick squashfs fix

Signed-off-by: Rajkumar Pennadam Ramamoorthy <rpennadamram@marvell.com>



#### Why I did it
MRVL prestera driver is not cloned from the correct commit.
#### How I did it
Update tag version in Marvell platform make file to pick correct commit-id.
#### How to verify it
clone the repo, trigger the build. Verify correct commit is cloned for driver compilation in path "sonic-buildimage/platform/marvell-armhf/prestera/mrvl-prestera"
#### Which release branch to backport (provide reason below if selected)




#### Description for the changelog

fixes #8479 

#### A picture of a cute animal (not mandatory but encouraged)
